pkgs/applications/misc/dstask/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "dstask";
+  version = "0.18";
+  rev = "d687a0f9aec634a808cfa0df4aecb06ab4f57732";
+
+  src = fetchFromGitHub {
+    owner = "naggie";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "16z5zlfj955pzsj0l58835slvpchdaq2vbyx2fjzi6y9xn1z2nd1";
+  };
+
+  vendorSha256 = null;
+
+  buildFlagsArray = [ ''-ldflags=-w -s
+    -X "github.com/naggie/dstask.VERSION=${version}"
```

I should have included this in the original suggestion. I made it needlessly difficult :sweat_smile: 
```suggestion
    -X "github.com/naggie/dstask.VERSION=v${version}"
```

---

Sorry about that, I missed the comment. Thank you very much for the help!